tests/slow/src/test/scala/tests/BloopPantsSuite.scala

```diff
@@ -263,4 +265,31 @@ class BloopPantsSuite extends FastpassSuite {
     )
   }
 
+  test("specify main_class") {
+    val workspace = Workspace(s"""$pantsIni
+                                 |/app/BUILD
+                                 |scala_library(
+                                 |  name="my-library",
+                                 |  sources=['my-library/**/*.scala']
+                                 |)
+                                 |jvm_binary(
+                                 |  name="my-binary",
+                                 |  dependencies=[':my-library'],
+                                 |  main="com.company.Main"
+                                 |)
+                                 |/app/my-library/com/company/Main.scala
+                                 |package com.company
+                                 |object Main { def main(args: Array[String]): Unit = () }
+                                 |""".stripMargin)
+    workspace.run("create" :: "--name" :: "test" :: "app::" :: Nil).succeeds
+    val projects = workspace.projects()
+    val binary = projects("app:my-binary")
+    binary.platform match {
+      case Some(Config.Platform.Jvm(_, Some(main), _, _)) =>
```

Comment:
I recommend using `jvm: Config.Platform.Jvm` instead of using `unapply` pattern matching. The compile errors are quite cryptic in case the upstream bloop-config module adds new fields to this class

Reply:
Updated, thanks!